src/SFA.DAS.ProviderApprenticeshipsService.Web/Orchestrators/Mappers/ApprenticeshipMapper.cs

```diff
@@ -59,7 +60,7 @@ public class ApprenticeshipMapper : IApprenticeshipMapper
             _academicYearValidator = academicYearValidator;
         }
 
-        public ApprenticeshipViewModel MapApprenticeship(Apprenticeship apprenticeship)
+        public ApprenticeshipViewModel MapApprenticeship(Apprenticeship apprenticeship, CommitmentView commitment)
```

Comment:
As discussed, instead of only fetching Apprenticeship to map to ApprenticeshipViewModel, this PR changes it to fetch the Apprenticeship and also the CommitmentView to feed into the mapping, but only 1 field from CommitmentView is actually used.

It was suggested, that Apprenticeship could be expanded to include the extra field, and that could be supplied from the commitments api, so we remain with a single call. However, Apprenticeship is also used in commands (as well as queries), and it would be confusing for the fields to be discarded during a command.

At some later point, we could rationalise the entities/dto's away from god-like objects and into a more fine grained menagerie, at which point we might create a single object to represent the data needed here.